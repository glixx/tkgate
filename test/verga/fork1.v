//: beginnetlist top
module top;
  reg [31:0] a,b,c;

  initial
    begin
	fork
	  @(a) $display("got a");
	  @(b) $display("got b");
	  @(c) $display("got c");
	join
	$display("done with fork");
    end

  initial
    begin
	#1 a = 1;
	#1 b = 1;
	#1 c = 1;
    end

endmodule
//:endnetlist top
