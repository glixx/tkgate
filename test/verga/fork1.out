got a
got b
got c
done with fork
